README.md

```diff
@@ -30,7 +30,12 @@ This repo aims to provide a stripped down version of the default [WooCommerce E2
 
 ### Setup
 
-Note: the following assumes an existing WooCommerce extension with an existing `package.json`. 
+**Requirements**
+
+* Note that `npm@6` is required for setup
+* The following steps assumes there is an existing WooCommerce extension with an existing `package.json`. If your project does not contain a `package.json` file, create one using `npm init`. More information on this can be found [here](https://docs.npmjs.com/creating-a-package-json-file#creating-a-new-packagejson-file).
```

Comment:
For people who scan vs read documentation/instructions it's better to have steps they will take on their own lines

> If your project does not contain a `package.json` file, create one:

```
npm init
```

> More information on ...

Reply:
Good suggestion, I'll break these out.